docs/changelog/overview.mdx

```diff
@@ -12,6 +12,15 @@ This page lists changes to Flare's API.
 Release notes for the Flare Platform can be found on the [product documentation website](https://docs.flare.io/releases).
 </Note>
 
+<Update label="2025-01" description="API - January 2025">
+  Released new [/firework/v2/activities/ <Icon icon="code" size={16} />](/api-reference/v2/endpoints/activities/get-activities-).
```

Comment:
Can you use the same formatting as the other changes? We put icons on single lines.

Reply:
We should avoid saying this is a new endpoint, in order to control expectations.

How about:
- Modified the get-activities endpoint to accept the UID as a query parameter instead of as path parameters. This avoids confusion that some users had when using uids that have multiple slashes. The old endpoint with path parameters was kept for backwards compatibility.